src/components/BookList.jsx

```diff
@@ -1,40 +1,35 @@
 /* eslint-disable linebreak-style */
+// BookList.js
+
 import React from 'react';
-import PropTypes from 'prop-types';
+import { useDispatch, useSelector } from 'react-redux';
+
+function BookList() {
+  const dispatch = useDispatch();
+  const books = useSelector((state) => state.books.books);
```

Comment:
You are trying to access `.books` on `state.books`, however, the final state of the books is the `state.books`. This is making the `const books` undefined and leading to no books being rendered on the UI. Kindly fix this error. ⬇️
![image](https://github.com/MasumaJaffery/bookstore/assets/33268927/3a2cd754-6d13-422e-9281-7adb50e08e81)